geoh5py/__init__.py

```diff
@@ -18,25 +18,6 @@
 # ''''''''''''''''''''''''''''''''''''''''''''''''''''''''''''''''''''''''''''''
 
 
-# flake8: noqa
-
 __version__ = "0.12.0a1"
 
```

Comment:
This is quite a commitment to change the init of geoh5py in this issue. 

Are we sure it's used nowhere?

Should be an issue on itself to test our packages